server/static/js/index/home_page_preloader.js

```diff
@@ -0,0 +1,31 @@
+"use strict";
+const DEVICE_PRELOADER = '#table_device_preloader';
+const USERS_PRELOADER = '#table_users_preloader';
+
+// The function is intended to execute the script when sending an AJAX request.
```

Comment:
No it isn't ^_^
This function simply triggers the loading spinner to be displayed on top of devices table. Please, reflect that in comment.
And yes - make it a proper JS docstring too.